video/vdu_sys.h

```diff
@@ -36,6 +37,11 @@ typedef union {
 // Wait for eZ80 to initialise
 //
 void VDUStreamProcessor::wait_eZ80() {
+	if(esp_reset_reason() == ESP_RST_SW) {
```

Comment:
This code is to skip the handshake of esp32 and ez80 when rebooted by the updater. This might be removed, because the update tool on ez80 side could mimic the handshake.